tests/foreman/ui/test_remoteexecution.py

```diff
@@ -553,3 +566,76 @@ def test_positive_matcher_field_highlight(session):
 
     :assignee: dsynk
     """
+
+
+@pytest.mark.tier3
+def test_positive_configure_cloud_connector(
+    session, default_sat, subscribe_satellite, fixture_enable_receptor_repos
+):
+    """Install Cloud Connector through WebUI button
+
+    :id: 67e45cfe-31bb-51a8-b88f-27918c68f32e
+
+    :Steps:
+
+        1. Navigate to Configure > Inventory Upload
+        2. Click Configure Cloud Connector
+        3. Open the started job and wait until it is finished
+
+    :expectedresults: The Cloud Connector has been installed and the service is running
+
+    :CaseLevel: Integration
+
+    :CaseComponent: RHCloud-CloudConnector
+
+    :CaseImportance: Medium
+
+    :assignee: lhellebr
+
+    :BZ: 1818076
+    """
+    # Copy foreman-proxy user's key to root@localhost user's authorized_keys
+    default_sat.add_rex_key(satellite=default_sat)
+
+    # Set Host parameter source_display_name to something random.
+    # To avoid 'name has already been taken' error when run multiple times
+    # on a machine with the same hostname.
+    host_id = Host.info({'name': default_sat.hostname})['id']
+    Host.set_parameter(
+        {'host-id': host_id, 'name': 'source_display_name', 'value': gen_string('alpha')}
+    )
+
+    with session:
+        if session.cloudinventory.is_cloud_connector_configured():
+            pytest.skip(
+                'Cloud Connector has already been configured on this system. '
+                'It is possible to reconfigure it but then the test would not really '
+                'check if everything is correctly configured from scratch. Skipping.'
+            )
+        session.cloudinventory.configure_cloud_connector()
```

Comment:
if you wanted to configure this from scratch why not adding in fixture (a session/module scope) ensure that will be ran only once in the any worker satellite.

Reply:
Because that's what I'm testing, I can't use fixture for the very thing the test should be testing.